UltraStar Play/Packages/playshared/Runtime/Util/Extensions/VisualElementExtensions.cs

```diff
@@ -170,4 +170,12 @@ public static List<VisualElement> GetAncestors(this VisualElement visualElement)
         }
         return ancestors;
     }
+
+    public static void AddNextTo(this VisualElement source, VisualElement target)
```

Comment:
- below `PlaceInFront` is called. This is more descriptive than `AddNextTo`, thus please call the method accordingly.
    - the name `AddNextTo` is not clear whether it is placed before/after the item